magprime/automated_emails.py

```diff
@@ -26,7 +26,7 @@ def __init__(self, event):
            lambda a: a.staffing and days_before(7, c.UBER_TAKEDOWN), sender='MAGFest Staff Suite <chefs@magfest.org>')
 
 AutomatedEmail(Attendee, 'MAGFest Staff Suite Volunteering', 'food_interest.txt',
-           lambda a: a.requested(c.FOOD_PREP) and not a.assigned_depts, sender='MAGFest Staff Suite <chefs@magfest.org>')
+           lambda a: a.requested(c.FOOD_PREP) or a.assigned_to(c.FOOD_PREP), sender='MAGFest Staff Suite <chefs@magfest.org>')
```

Comment:
This is a change in who the email would actually get sent to.  The original version was "send to everyone who requested food prep but who is not yet assigned anywhere".  Now it's "send to everyone who either requested food prep or who has already been assigned there".

This might be a good change, since if I recall correctly that email was originally configured for last year before we assigned several months worth of volunteers.  I just want to make sure that the change was intentional.